src/components/PDFThumbnail/index.native.tsx

```diff
@@ -1,19 +1,25 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {View} from 'react-native';
 import Pdf from 'react-native-pdf';
 import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import addEncryptedAuthTokenToURL from '@libs/addEncryptedAuthTokenToURL';
+import variables from '@styles/variables';
 import type PDFThumbnailProps from './types';
 
-function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, enabled = true, onPassword}: PDFThumbnailProps) {
+function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, enabled = true, onPassword, onLoadError}: PDFThumbnailProps) {
     const styles = useThemeStyles();
+    const theme = useTheme();
     const sizeStyles = [styles.w100, styles.h100];
+    const [hasError, setHasError] = useState(false);
 
     return (
         <View style={[style, styles.overflowHidden]}>
-            <View style={[sizeStyles, styles.alignItemsCenter, styles.justifyContentCenter]}>
-                {enabled && (
+            <View style={[sizeStyles, !hasError && styles.alignItemsCenter, styles.justifyContentCenter]}>
+                {enabled && !hasError && (
```

Comment:
Is there any reason you added different conditions when rendering the PDF component and thumbnail?


Why are we checking the hasError in native only and not other platforms?

https://github.com/Expensify/App/blob/31caba2c668877e26dc40a8d91c17d369df24403/src/components/PDFThumbnail/index.native.tsx#L22

https://github.com/Expensify/App/blob/31caba2c668877e26dc40a8d91c17d369df24403/src/components/PDFThumbnail/index.tsx#L57